test/TestHelper.hs

```diff
@@ -0,0 +1,81 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE QuasiQuotes #-}
+module TestHelper
+    ( withApp
+    , module X
+    ) where
+
+-- Application code
+import Model as X
+import Model.Comment as X
+import Model.User as X
+import Model.UserComment as X
+import Foundation as X
+
+import Application (makeFoundation)
+import Settings (Extra, parseExtra)
+
+-- Useful libraries
+import Control.Applicative as X
+import Control.Monad as X
+import Data.Text as X (Text)
+import Data.Aeson as X
+import Data.Monoid as X
+import Database.Persist as X hiding (get, delete)
+
+import Control.Monad.IO.Class (MonadIO)
+import Control.Monad.Reader (ReaderT, ask)
+import Database.Persist.Sql
+    ( SqlBackend
+    , rawExecute
+    , rawSql
+    , unSingle
+    , connEscapeName
+    )
+import Text.Shakespeare.Text (st)
+import Yesod.Default.Config
+
+-- Test framework
+import Test.Hspec as X
+import Yesod.Test as X
+
+-- Test helpers
+import TestHelpers.DB as X
+import TestHelpers.Request as X
+import TestHelpers.Assertions as X
+import TestHelpers.Auth as X
+
+import qualified Data.Text as T
+
+withApp :: SpecWith App -> Spec
+withApp = before $ do
```

Comment:
Should be try to maintain some level of formatting parity with the current scaffold?

https://github.com/yesodweb/yesod-scaffold/blob/postgres/test/TestImport.hs

I'm leaning no. Partly because we have to change some things, and partly because I'm not sure the value.